org.librarysimplified.http.vanilla/src/main/java/org/librarysimplified/http/vanilla/internal/LSHTTPRequest.kt

```diff
@@ -9,7 +9,7 @@ import org.librarysimplified.http.api.LSHTTPResponseType
 
 class LSHTTPRequest(
   private val client: LSHTTPClient,
-  private val allowRedirects: AllowRedirects,
+  val allowRedirects: AllowRedirects,
```

Comment:
It's actually not truly public, because users only ever see the `LSHTTPRequestType` interface and not raw `LSHTTPRequest` values. It could be reduced to `internal` though, so I'll do that.

Reply:
Same question about internal! I can't see any new use of the field.

Reply:
Actually, I see what you mean. I just realized this was left over from when the redirect handling code checked to see how the redirect behaviour was specified. It no longer does that, so it no longer needs access to this field.